go-controller/pkg/ovn/gateway_test.go

```diff
@@ -31,6 +33,33 @@ func init() {
 	format.MaxLength = 0
 }
 
+func generateAdvertisedUDNIsolationExpectedNB(testData []libovsdbtest.TestData, networkName string, clusterIPSubnets []*net.IPNet, nodeSwitch *nbdb.LogicalSwitch, addrSet addressset.AddressSet) []libovsdbtest.TestData {
```

Comment:
is this a duplicate of what we are doing in the real code?

Reply:
more or less yeah, I need this to ensure the ACLs are as expected in the fake db.